tests/_benderjs/ckeditor/static/tools.js

```diff
@@ -91,6 +91,10 @@
 			return bender.tools.fixHtml( html, stripLineBreaks );
 		},
 
+		isBuild: function() {
```

Comment:
There is a much better namespace for this function: `bender.tools.env` namsepace. We should avoid throwing everything into `bender.tools` as we'd quickly have too much members there.

To better integrate with existing code over there, you can see that there are just properties. We don't expect build type (dev/build) to change between the tests, so it could be a property just like `bender.tools.env.mobile`.